src/libbson/doc/bson_array_as_json.rst

```diff
@@ -22,7 +22,7 @@ Description
 
 The :symbol:`bson_array_as_json()` function shall encode ``bson`` as a UTF-8
 string using libbson's legacy JSON format, except the outermost element is
-encoded as JSON array, rather then JSON document.
+encoded as JSON array, rather than JSON document.
```

Comment:
"as a JSON array, rather than a JSON document"

Reply:
adding the "as" is certainly better

Reply:
OK!